hapi/src/services/hyperion/index.js

```diff
@@ -3,10 +3,12 @@ const moment = require('moment')
 const updaters = require('./updaters')
 const hyperionStateService = require('../../gql/hyperion-state.gql')
 const { edenElectionGql } = require('../../gql')
-const { hyperionConfig } = require('../../config')
-const { hasuraUtil, axiosUtil, sleepUtil, isEdenExpense } = require('../../utils')
+const { hyperionConfig, eosConfig } = require('../../config')
+const { hasuraUtil, axiosUtil, sleepUtil } = require('../../utils')
 
 const TIME_BEFORE_IRREVERSIBILITY = 164
+let   LAST_CONSULTED_DAY          = null
+let   LAST_CONSULTED_DATA         = null
```

Comment:
why empty space?

Reply:
what data is being consulted? maybe we can find a better name for this variable